lib/recurly/resource.rb

```diff
@@ -127,7 +127,7 @@ class Resource
     #     e.message # => "Can't find Account with account_code = tortuga"
     #   end
     class NotFound < API::NotFound
-      def initialize message
+      def initialize(message)
```

Comment:
This is great, but is there any way you could put these changes in their own commit?